docker/centos7

```diff
@@ -0,0 +1,26 @@
+FROM centos:7
+
+ARG ANTARES_DEPS_VERSION
+
+# Install requirements : update repo
+RUN yum update -y
+
+# Install requirements : install epel
+RUN yum install -y epel-release 
+
+# Install requirements
+RUN yum install -y git redhat-lsb-core gcc gcc-c++ make wget centos-release-scl scl-utils rpm-build &&\
+    yum install -y cmake3 devtoolset-7
```

Comment:
Only one `yum` command ?

Reply:
We need to have 2 command because `devtoolset-7` need `centos-release-scl` to be installed first.